CONTRIBUTING.md

```diff
@@ -192,7 +192,7 @@ Each commit message consists of a **header**, a **body** and a **footer**.  The
 format that includes a **type**, a **scope** and a **subject**:
 
 ```
-<type>(<scope>): <subject>
+(revert: )<type>(<scope>): <subject>
```

Comment:
Oh right. I was thinking the scope is actually optional. Maybe we should note what's optional what's not?